Assets/Scripts/FlagManager.cs

```diff
@@ -3,40 +3,32 @@
 using UnityEngine;
 
 public class FlagManager : MonoBehaviour {
-	[SerializeField]
-	private int flagQuantity;
-	[SerializeField]
-	private GameObject[] spawnLocations;
+
 	[SerializeField]
 	private GameObject flagPrefab;
 
-	private GameObject[] flags;
-	private int[] spawnOccupied;
+    private GameObject[] spawnLocations;
+    private List<GameObject> flags;
```

Comment:
I'll look at that this afternoon, wasn't sure how C# handled dynamic sized arrays.

Reply:
I'm gonna stick with the List, it gives me the nicety of being able to remove it from the list below in this class.  Otherwise I would have to write code to do that.